### PR TITLE
team snippets improvements, `about` article punctuation correction

### PR DESCRIPTION
**Pull request by orangesunny**
## Proposed changes

team snippets were rewritten for better language and readability.

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [x] I have squashed my commits into logic units.
- [x] I have described the changes in the commit messages.

### CODE REVIEW

**Comment by orangesunny** [comment]
@comradekingu 
Do you want to share your opinion on the snippets?

File: weblate_web/templates/about.html [hunk 1]
```diff
@@ -51,23 +51,23 @@ <h2 class="section-title crypto">{% trans "Your Weblate Team" %}</h2>
                      <div class="content-p">
                        <p>Michal Čihař</p>
                        <br>
-                       <t>{% trans "Master of the idea and code. Leading the development and custom or large-scale solutions." %}</t>
+                       <t>{% trans "Author of the idea; code and development leader. He’s also designing custom and large-scale solutions." %}</t>
```
Comment (comradekingu) [inline weblate_web/templates/about.html:54]:
```suggestion
                       <t>{% trans "Author and implementation. Designs custom and large-scale solutions." %}</t>
```

File: weblate_web/templates/about.html [hunk 2]
```diff
@@ -51,23 +51,23 @@ <h2 class="section-title crypto">{% trans "Your Weblate Team" %}</h2>
...
                     </div>
                 </div>
                 <div class="weblate-team-content">
                     <img src="https://hosted.weblate.org/avatar/80/orangesunny.png" alt="Benjamin Alan Jamie" />
                     <div class="content-p">
                         <p>Benjamin Alan Jamie</p>
                         <br>
-                        <t>{% trans "Community manager and support guru, helping with smooth workflows. The one to talk sales with." %}</t>
+                        <t>{% trans "Community Manager and support guru who’s helping you with smooth workflows. Our salesperson." %}</t>
```
Comment (comradekingu) [inline weblate_web/templates/about.html:62]:
```suggestion
                        <t>{% trans "Community support manager helping smooth out your workflow. Our salesperson." %}</t>
```

File: weblate_web/templates/about.html [hunk 3]
```diff
@@ -51,23 +51,23 @@ <h2 class="section-title crypto">{% trans "Your Weblate Team" %}</h2>
...
                     </div>
                     </div>
                     <div class="weblate-team-content">
                         <img src="https://hosted.weblate.org/avatar/80/vitavalka.png" alt="Víťa Válka" />
                         <div class="content-p">
                             <p>Víťa Válka</p>
                             <br>
-                            <t>{% trans "Interface and graphics designer elevating user happiness. Continually optimizing Weblate." %}</t>
+                            <t>{% trans "Interface and graphics designer elevating user happiness. He’s continually optimizing Weblate." %}</t>
```
Comment (comradekingu) [inline weblate_web/templates/about.html:70]:
```suggestion
                            <t>{% trans "Interface and graphics designer evaluating user happiness. Continually optimizing perception." %}</t>
```

**Comment by comradekingu** [comment]
@orangesunny
I see some of what you are trying to do. Give me some time to sleep on it and I think I can solve it better than my preliminary proposal here. I can't get into the minutia, but I can present something that works, hopefully.

**Comment by orangesunny** [comment]
> @orangesunny
> I see some of what you are trying to do. Give me some time to sleep on it and I think I can solve it better than my preliminary proposal here. I can't get into the minutia, but I can present something that works, hopefully.

Thanks, for sure, no hurry. 

I believe that my changes are better than what’s live now (written quickly some time ago, not much fluent or human-feel text :D).
I am trying to make it smoother and more lifeful and casual, not shorted.

File: weblate_web/templates/about.html (hunk 1 above)
Comment (orangesunny) [reply weblate_web/templates/about.html:54]: not this change

File: weblate_web/templates/about.html (hunk 2 above)
Comment (orangesunny) [reply weblate_web/templates/about.html:62]: considered

File: weblate_web/templates/about.html (hunk 3 above)
Comment (orangesunny) [reply weblate_web/templates/about.html:70]: not this change

**Pull request merged**